src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs

```diff
@@ -169,6 +169,24 @@ protected CodeFixTest()
         /// <returns>The <see cref="CodeFixProvider"/> to be used.</returns>
         protected abstract IEnumerable<CodeFixProvider> GetCodeFixProviders();
 
+        /// <inheritdoc />
+        protected override bool IsCompilerDiagnosticIncluded(Diagnostic diagnostic, CompilerDiagnostics compilerDiagnostics)
+        {
+            if (base.IsCompilerDiagnosticIncluded(diagnostic, compilerDiagnostics))
+            {
+                return true;
+            }
+
+            return CodeFixProvidersHandleDiagnostic(diagnostic);
```

Comment:
Why does this need to be a local function?

Reply:
It doesn't need to be, but it's not hurting anything either. I merged so @taori can start using this 😄

Reply:
Original function combined multiple local functions. Should have refactored that one out too, like other methods.